src/abstractdataframe/show.jl

```diff
@@ -165,11 +165,6 @@ function getprintedwidth(maxwidths::Vector{Int}) # -> Int
     return totalwidth
 end
 
-# For Julia < 0.5
-if VERSION < v"0.5.0-dev+2023"
```

Comment:
That function doesn't exist in Compat, only the one without any arguments (I didn't include it there since it's a bit a cheating: `io` isn't used at all).